internal/server/iigointernal/base_president.go

```diff
@@ -139,6 +151,7 @@ func (p *basePresident) getAllocationRequests(commonPool int) map[int]int {
 	return result
```

Comment:
Yup, the error is there since we need this base implementation to satisfy the interface function signature, but our base implementation itself doesn't return an error ever.